package.json

```
@@ -3,8 +3,8 @@
   "private": true,
   "version": "0.0.1",
   "scripts": {
-    "audit": "npm audit && lerna exec --concurrency 1 'echo $LERNA_PACKAGE_NAME && npm audit'",
-    "audit:fix": "npm audit fix && lerna exec --concurrency 1 'echo $LERNA_PACKAGE_NAME && npm audit fix'",
+    "audit": "npm audit & lerna exec 'echo $LERNA_PACKAGE_NAME & npm audit'",
+    "audit:fix": "npm audit fix & lerna exec 'echo $LERNA_PACKAGE_NAME & npm audit fix'",
```

```suggestion
    "audit:fix": "npm audit fix --production & lerna exec 'echo $LERNA_PACKAGE_NAME & npm audit fix --production'",
```
fix doesn't return errors, so we don't need `--no-bail`